### DISCUSSION TITLE
ESP32 Feeezes

### DISCUSSION THREAD

**Discussion by MrGibbage** [ESPresense/ESPresense#430] [Q&A]
I have this ESP32 running ESPresense 2.1.0
https://www.amazon.com/gp/product/B08246MCL5

I have seen that it typically can't run more than a day without freezing. Normally the blue light blinks all the time, but when it freezes, I have seen that the blue light will be on but not blinking. I believe I have also seen it freeze with the blue light off, if that helps. When it does freeze there is no MQTT traffic out of the ESP32 at all.

Any ideas? 

Skip

**Comment by DTTerastar** [comment]
The best thing would be to hook up something to the serial monitor and capture all the log output.  The esp32 should automatically reset itself if something goes wrong.  I have seen that fail once or twice, but never consistently.  We should be able to tell what's going on if you have the serial output.

**Reply by gadric** [reply]
Have something the same issues. But it is highly inconsistent. Somethings it goes 10 days without any issue. somtimes only 2 days.
